pkg/pkg.mk

```diff
@@ -13,6 +13,7 @@ GIT_APPLY_PATCHES:=if test -d "$(PKG_DIR)"/patches; then \
 	fi
 
 $(PKG_BUILDDIR)/.git-downloaded:
+	rm -Rf $(PKG_BUILDDIR)
```

Comment:
>> Would have bet, that the -p in the mkdir below has the same effect
>
> On my system, `mkdir -p` has luckily not the same effect as `rm -Rf`

I meant the combination of `rm -Rf ${dir} && mkdir -p ${dir}` vs. just `mkdir -p ${dir}` at least for the scope of what we want to achieve (don't fail because directory already exists) ;-)